src/sentry/issues/utils.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, Mapping
+
+from sentry.issues.grouptype import GroupCategory, get_group_types_by_category
+from sentry.issues.issue_occurrence import IssueOccurrence
+from sentry.models import Project
+
+
+def skip_group_processing(
+    occurrence: IssueOccurrence, options: Mapping[str, Any], project: Project
+) -> bool:
+    return (
+        occurrence.type.type_id in get_group_types_by_category(GroupCategory.PERFORMANCE.value)
+        and options.get("performance.issues.send_to_issues_platform", False)
+        and project.get_option("sentry:performance_issue_send_to_issues_platform", False)
```

Comment:
Are these backwards? This function should return `True` when we should skip group processing, so should it be:
`is_performance and (not option_1 and not option_2)`?

I think that when this is `False` we also need to make sure that we're not creating groups via the old method.

Reply:
yeah I think I need two separate flags here - to send occurrences to the platform (this will be tied to skipping group processing) and create a group. So we can first enable writing occurrences and then separately roll out group creation.